crates/rolldown_error/src/lib.rs

```diff
@@ -16,3 +18,8 @@ pub use crate::{
   type_aliases::{DiagnosableResult, UnhandleableResult},
   types::diagnostic_options::DiagnosticOptions,
 };
+
+fn _usage_should_able_to_auto_convert_outside_errors() -> DiagnosableResult<()> {
```

Comment:
what is this used for?

Reply:
It's used to ensure behavior is valid. 

It's common seen pattern in rust. 
https://github.com/rolldown/rolldown/blob/7a4d36b04af6f750492f727518d05de67d64693c/crates/rolldown/src/bundler.rs#L238